src/main/java/apoc/load/Jdbc.java

```diff
@@ -221,11 +226,23 @@ public boolean hasNext() {
         }
 
         private Object convert(Object value) {
+
             if (value instanceof UUID || value instanceof BigInteger || value instanceof BigDecimal) {
                 return value.toString();
             }
             if (value instanceof java.util.Date) {
-                return ((java.util.Date) value).getTime();
+                if(!config.isEmpty()){
+                    LoadJdbcConfig loadJdbcConfig = new LoadJdbcConfig(config);
```

Comment:
Shouldn't this be handled upfront? The JDBC config created etc? And instead of the many ifs an enum or lambda would be better suited to be passed in as "DateTransformer" or so.